testapi.pm

```diff
@@ -1954,9 +1862,7 @@ if C<$timeout> timeout is hit. Default timeout is 60s.
 
 =cut
 
-sub assert_shutdown {
-    my ($timeout) = @_;
-    $timeout //= ONE_MINUTE;
+sub assert_shutdown ($timeout = ONE_MINUTE) {
```

Comment:
```suggestion
sub assert_shutdown ($timeout = undef) {
    $timeout //= ONE_MINUTE;
```

Reply:
can you explain, why again?

Reply:
ok, I get it. To keep the behaviour completely unchanged I need the default parameter *and* the default value if `$timeout` is undef because otherwise `assert_shutdown(undef)` would behave different and actually end up with no valid value in the end.